src/Gui/Utilities.h

```diff
@@ -104,6 +104,17 @@ struct vec_traits<App::Color> {
 private:
     const vec_type& v;
 };
+
+template <>
+inline SbMatrix convertTo<SbMatrix, Base::Matrix4D>(const Base::Matrix4D& vec2)

```

Comment:
A bit nit-picking but a matrix shouldn't be called vec2.
And for consistency reasons a converter from SbMatrix to Base::Matrix4D should be available, too.

Reply:
Yeah, I don't like the `vec2` too but linter complains if the template specialization uses different name :/ I'll provide other side of conversion.

Reply:
Oh, OK. Then leave it.